src/components/secret-display/states.ts

```diff
@@ -0,0 +1,8 @@
+export const SecretDisplayStates = {
+  loading: 'loading',
+  revealed: 'revealed',
+  noSeed: 'noSeed',
+  securedWithBiometrics: 'securedWithBiometrics',
+} as const;
+
+export type SecretDisplayStatesType = typeof SecretDisplayStates[keyof typeof SecretDisplayStates];
```

Comment:
We should talk as a team about enums :P I think enum would be fine here but probably nbd

Reply:
I like enums, but I was trying to follow the TypeScript style guide.